### PR TITLE
✨ Added dbus-send query for playback status (spotify)

### PR DESCRIPTION
**Pull request by HalbardHobby**

### CODE REVIEW

**Comment by erikw** [comment]
Thank you! Just one thing to fix

**Comment by HalbardHobby** [comment]
Just tell me to fix the PR.

**Comment by HalbardHobby** [comment]
Still waiting the fix request 😅😅😅

**Comment by erikw** [comment]
Please go ahead and fix it =)

**Comment by HalbardHobby** [comment]
I meant that you didn't tell what I need to fix 😅

**Review by erikw — CHANGES_REQUESTED** [review]

**Comment by erikw** [comment]
Ah now your comment makes more sense. I'm sorry, I had forgotten to publish my comment, it was still in draft mode haha

File: segments/now_playing.sh [hunk 1]
```diff
@@ -12,11 +12,12 @@ TMUX_POWERLINE_SEG_NOW_PLAYING_MPD_SIMPLE_FORMAT_DEFAULT="%artist% - %title%"
 TMUX_POWERLINE_SEG_NOW_PLAYING_PLAYERCTL_FORMAT_DEFAULT="{{ artist }} - {{ title }}"
 TMUX_POWERLINE_SEG_NOW_PLAYING_RHYTHMBOX_FORMAT_DEFAULT="%aa - %tt"
 TMUX_POWERLINE_SEG_NOW_PLAYING_NOTE_CHAR_DEFAULT="♫"
+# Music player to use. Can be any of {audacious, banshee, cmus, itunes, lastfm, mocp, mpd, mpd_simple, pithos, playerctl, rdio, rhythmbox, spotify, spotify_wine, file}.
```
[the remark this answers is not in the document]
Comment (HalbardHobby) [reply segments/now_playing.sh]: Moved back to place in d95c27e 😁

**Comment by HalbardHobby** [comment]
No worries, already fixed the issue.

**Pull request merged**